tests/integration/jobs/jobs.spec.ts

```diff
@@ -150,4 +151,271 @@ describe('job', function () {
+        } satisfies components['schemas']['createJobPayload'];
+
+        const createJobResponse = await requestSender.createJob({
+          requestBody: requestBody,
+        });
+
+        if (createJobResponse.status !== StatusCodes.CREATED) {
+          throw new Error();
+        }
+
+        const createdJobId = createJobResponse.body.id;
+
+        const setPriorityResponse = await requestSender.updateJobPriority({
+          pathParams: { jobId: createdJobId },
+          requestBody: { priority: 'VERY_LOW' },
+        });
+
+        expect(setPriorityResponse).toSatisfyApiSpec();
+        expect(setPriorityResponse).toMatchObject({
+          status: httpStatusCodes.NO_CONTENT,
+          // eslint-disable-next-line @typescript-eslint/naming-convention
+          headers: { 'x-status-reason': 'Cannot update priority that equals to the current priority' },
+        });
+      });
+    });
+
+    describe('Bad Path', function () {
+      it('The system should return a 404 status code along with a specific validation error message detailing the non exists job', async function () {
+        const getJobResponse = await requestSender.updateJobPriority({ pathParams: { jobId: jobId }, requestBody: { priority: 'VERY_HIGH' } });
+
+        expect(getJobResponse).toSatisfyApiSpec();
+        expect(getJobResponse).toMatchObject({
+          status: httpStatusCodes.NOT_FOUND,
+          body: { message: 'JOB_NOT_FOUND' },
+        });
+      });
+    });
+
+    describe('Sad Path', function () {
+      it('should return 500 status code when the database is down on create job', async function () {
+        jest.spyOn(prisma.job, 'findUnique').mockRejectedValueOnce(new Error('Database error'));
+
+        const response = await requestSender.updateJobPriority({ pathParams: { jobId: jobId }, requestBody: { priority: 'VERY_HIGH' } });
+
+        expect(response).toSatisfyApiSpec();
+        expect(response).toMatchObject({ status: httpStatusCodes.INTERNAL_SERVER_ERROR, body: { message: 'Database error' } });
+      });
+    });
+  });
+
+  describe('#updateStatus', function () {
+    describe('Happy Path', function () {
+      it("should return 201 status code and modify job's status", async function () {
+        const requestBody = {
+          name: 'DEFAULT',
+          creator: 'UNKNOWN',
+          data: { stages: [] },
+          type: 'PRE_DEFINED',
+          notifications: {},
+          userMetadata: {},
+          priority: 'VERY_LOW',
+        } satisfies components['schemas']['createJobPayload'];
+
+        const createJobResponse = await requestSender.createJob({
+          requestBody: requestBody,
+        });
+
+        if (createJobResponse.status !== StatusCodes.CREATED) {
+          throw new Error();
+        }
+
+        const createdJobId = createJobResponse.body.id;
+
+        const setStatusResponse = await requestSender.updateStatus({ pathParams: { jobId: createdJobId }, requestBody: { status: 'COMPLETED' } });
```

Comment:
you need to make sure this returns status code 200

Reply:
done